apps/preview/next/package.json

```diff
@@ -8,34 +8,20 @@
     "build:next": "yarn build",
     "serve": "next start",
     "lint": "next lint --ext .tsx,.ts",
-    "lint:fix": "next lint --fix",
-    "clean": "rimraf ./sfui",
-    "clean-coverage": "rimraf ./coverage && rimraf ./.nyc_output && rimraf ../../../packages/sfui/frameworks/react/coverage && rimraf ../../../packages/sfui/frameworks/react/.nyc_output",
-    "display-coverage": "yarn nyc report --cwd ../../../packages/sfui/frameworks/react --reporter=text --reporter=lcov",
-    "prepare-coverage-report": "frs-replace \"/apps/preview/next/sfui/react/\" \"/packages/sfui/frameworks/react/\" -i ./.nyc_output/out.json -o ../../../packages/sfui/frameworks/react/.nyc_output/out.json",
-    "sync-test": "cpy . ../../apps/preview/next/sfui/tests --cwd=../../../packages/tests/",
-    "sync-components": "cpy . !dist ../../../../apps/preview/next/sfui/react --cwd=../../../packages/sfui/frameworks/react/",
-    "sync:watch-test": "chokidar --silent \"../../../packages/tests/**/*.{cy.tsx,tsx,ts,png,jpg,jpeg,json}\" -c \"yarn sync-test\"",
-    "test": "TEST=true && yarn clean && yarn sync-test && concurrently \"yarn sync:watch-test\" \"yarn cypress open --component\"",
-    "test:ci": "TEST=true && yarn clean && yarn clean-coverage && yarn sync-test && yarn sync-components && CYPRESS_COVERAGE=true yarn percy exec -- cypress run --component && yarn prepare-coverage-report && yarn display-coverage",
-    "test:ci:react": "yarn test:ci"
+    "lint:fix": "next lint --fix"
   },
   "dependencies": {
-    "@storefront-ui/preview-shared": "workspace:^",
+    "@storefront-ui/preview-shared": "workspace:*",
     "classnames": "^2.3.2",
     "lodash-es": "^4.17.21",
-    "next": "^13.1.6",
+    "next": "latest",
```

Comment:
Not sure about this change. What if they make some really breaking changes in v14? Our CI will stop working immediately

I'd suggest to keep the ^13 version

Reply:
good point https://github.com/vuestorefront/storefront-ui/pull/2634/commits/e69761c4dd99511bb340f3a3a39fd6f190934071